### PR TITLE
Changes docs to point to cordapp-tutorial instead of cordapp-template where appropriate.

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

**Comment by joeldudleyr3** [comment]
Please do not include these changes in M8.

File: docs/source/tutorial-cordapp.rst [hunk 1]
```diff
@@ -4,14 +4,18 @@
...
 
-The CorDapp template
-====================
+The example CorDapp
+===================
 
-This guide covers how to get started with the `cordapp-template`. Please note there are two Corda repositories:
+.. note:: The example CorDapp was previously hosted in the `cordapp-template <https://github.com/corda/cordapp-template>`_ repository. It has now been 
+   moved into a new `cordapp-tutorial <https://github.com/corda/cordapp-tutorial>`_ repository. Going forward, the cordapp-template repo will contain an 
+   actual CorDapp template.
```
Comment (roger3cev) [inline docs/source/tutorial-cordapp.rst]: which should be used as the basis for CorDapp projects going forward?

**Comment by joeldudleyr3** [comment]
Also, this shouldn't be merged before @Clintonio 's fix to runnodes in cordapp-tutorial.

File: docs/source/tutorial-cordapp.rst [hunk 2]
```diff
@@ -440,32 +443,32 @@ served are as follows:
...
-**Submitting a purchase order via HTTP API:**
+**Submitting an IOU via HTTP API:**
 
-To create a purchase order from NodeA to NodeB, use:
+To create an IOU from NodeA to NodeB, use:
 
 .. sourcecode:: bash
 
-  echo '{"orderNumber": "1","deliveryDate": "2018-09-15","deliveryAddress": {"city": "London","country": "UK"},"items" : [{"name": "widget","amount": "3"},{"name": "thing","amount": "4"}]}' | cURL -T - -H 'Content-Type: application/json' http://localhost:10005/api/example/NodeB/create-purchase-order
+  echo '{"orderNumber": "1","deliveryDate": "2018-09-15","deliveryAddress": {"city": "London","country": "UK"},"items" : [{"name": "widget","amount": "3"},{"name": "thing","amount": "4"}]}' | cURL -T - -H 'Content-Type: application/json' http://localhost:10005/api/example/NodeB/create-iou
```
Comment (roger3cev) [inline docs/source/tutorial-cordapp.rst]: do we need delivery address for an IOU? is this the correct schema for a JSON object that will deserialise into an IOU object?

**Pull request merged**